tests/main/abort/task.yaml

```diff
@@ -17,7 +17,7 @@ execute: |
     echo "Abort with valid id - error"
     subdirPath="/snap/$SNAP_NAME/current/foo"
     mkdir -p $subdirPath
-    if snap install $SNAP_NAME; then
+    if . $TESTSLIB/install.sh $SNAP_NAME; then
```

Comment:
This is changing, but FWIW note that this initial dot isn't necessary. This tells bash to source the definitions inside the file into the local namespace, which isn't wanted for that case.